### PR TITLE
Fix a couple pad tests

### PR DESCRIPTION
**Pull request by cheeetoo**

### CODE REVIEW

File: extra/onnx_ops.py [hunk 1]
```diff
@@ -66,10 +66,10 @@ def _padding(X, pads=None, auto_pad="NOTSET", axes=None, constant_value=0.):
   constant_padder = Tensor(np.pad(np.zeros(X.shape), np_pads, constant_values=constant_value), dtype=X.dtype)
   return zero_padded + constant_padder
 
-def Pad(x: Tensor, pads: Tensor, constant_value: Tensor=None, axes: Tensor=None, mode="constant"):
+def Pad(x: Tensor, pads: Tensor, constant_value: Tensor=None, axes: Tensor=None, mode="constant", value: float=0.):
   assert mode == "constant"
-  constant_value = 0. if constant_value is None else constant_value.numpy()
-  seq_pads = pads.numpy().astype(np.int32).tolist()
+  constant_value = value if constant_value is None else constant_value.numpy()
+  seq_pads = pads if isinstance(pads, tuple) else pads.numpy().astype(np.int32).tolist()
```
Comment (geohot) [inline extra/onnx_ops.py:72]: Can you make type match?

Comment (cheeetoo) [reply extra/onnx_ops.py]: I changed it to convert pads to a list. Is that what you were looking for?

Comment (geohot) [reply extra/onnx_ops.py]: No, I mean the function type signature.

**Comment by geohot** [comment]
For all onnx test fixing PRs, can you include a list of the tests it fixes?

**Comment by cheeetoo** [comment]
Fixes:
- test_ConstantPad2d_cpu
- test_ZeroPad2d_cpu

File: extra/onnx_ops.py (hunk 1 above)
Comment (cheeetoo) [reply extra/onnx_ops.py]: Alright, fixed.

File: extra/onnx_ops.py [hunk 2]
```diff
@@ -66,10 +66,10 @@ def _padding(X, pads=None, auto_pad="NOTSET", axes=None, constant_value=0.):
   constant_padder = Tensor(np.pad(np.zeros(X.shape), np_pads, constant_values=constant_value), dtype=X.dtype)
   return zero_padded + constant_padder
 
-def Pad(x: Tensor, pads: Tensor, constant_value: Tensor=None, axes: Tensor=None, mode="constant"):
+def Pad(x: Tensor, pads: Tensor | tuple, constant_value: Tensor=None, axes: Tensor=None, mode="constant", value: float=0.):
```
Comment (geohot) [inline extra/onnx_ops.py:69]: Cool, I didn't know you could do this!

Comment (er-eis) [reply extra/onnx_ops.py]: [new feature in `3.10`](https://docs.python.org/3/whatsnew/3.10.html#pep-604-new-type-union-operator). unless we're ready to bump v from `3.8`->`3.10`, we should use `Union` instead.

**Pull request merged**